src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.IncrementalAnalyzerProcessor.cs

```diff
@@ -284,20 +356,8 @@ private static SyntaxNode GetMemberNode(ISyntaxFactsService service, SyntaxNode
                     return service.IsMethodLevelMember(memberNode) ? memberNode : null;
                 }
 
-                internal ProjectId GetActiveProject()
-                {
-                    ProjectId activeProjectId = null;
-                    if (_documentTracker != null)
-                    {
-                        var activeDocument = _documentTracker.TryGetActiveDocument();
-                        if (activeDocument != null)
-                        {
-                            activeProjectId = activeDocument.ProjectId;
-                        }
-                    }
-
-                    return null;
-                }
+                internal ProjectId? GetActiveProject()
+                    => _documentTracker?.TryGetActiveDocument()?.ProjectId;
```

Comment:
👍 

Note: why is _documentTracker ever null?

Reply:
This is null in OOP case. I tried to remove the null checks here (and also at another place in the coordinator) and got a remote NRE for null document tracking service.

Reply:
Bleagh.  Can you file issue taht OOP is busted.  We should ahve a DefaultIDocumentTracker there.

note: i ran into the same busted stuff in trying to clean up our persistence layer.  OOP has all the workspace stuff but doesn't MEF in a sane way afaict.

Reply:
Filed https://github.com/dotnet/roslyn/issues/39781